### ISSUE TITLE
Course export/restore

### ISSUE THREAD

**Issue by jmontoyaa** [chamilo/chamilo-lms#3796]
Allow resources to be exported/restored

**Comment by ywarnier** [comment]
I believe this now works in 2.0. @christianbeeznest ?

**Comment by christianbeeznest** [comment]
Yes, this has been verified as part of the course maintenance work in Improve backup/restore #6958. Resource export and restore are now working in 2.0
